Tutorials/jax.ipynb

```diff
@@ -173,23 +184,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "0 9.97-0.01j ± 0.087 [var=6.4e+00, R_hat=0.9993]\n",
-      "50 -3.2+0.1j ± 0.12 [var=7.9e+00, R_hat=1.0005]\n",
-      "100 -9.68-0.05j ± 0.064 [var=2.7e+00, R_hat=1.0007]\n",
-      "150 -10.65-0.02j ± 0.038 [var=1.0e+00, R_hat=1.0016]\n",
-      "200 -10.94-0.01j ± 0.027 [var=8.8e-01, R_hat=1.0002]\n",
-      "250 -11.08-0.00j ± 0.023 [var=6.0e-01, R_hat=0.9991]\n",
-      "300 -11.20+0.01j ± 0.024 [var=4.6e-01, R_hat=0.9992]\n",
-      "350 -11.21-0.00j ± 0.017 [var=2.9e-01, R_hat=0.9992]\n",
-      "400 -11.26+0.00j ± 0.014 [var=1.8e-01, R_hat=0.9997]\n",
-      "450 -11.28+0.00j ± 0.012 [var=1.3e-01, R_hat=0.9994]\n"
+      "0 5.36 ± 0.15 [σ²=15.31, R̂=0.9999]\n",
+      "50 -2.92862 ± 0.00027 [σ²=0.00000, R̂=1.4135]\n",
+      "100 -2.975733 ± 0.000024 [σ²=0.000000, R̂=1.4135]\n",
+      "150 -2.979545 ± 0.000018 [σ²=0.000000, R̂=1.4135]\n",
+      "200 -2.983247 ± 0.000017 [σ²=0.000000, R̂=1.4135]\n",
+      "250 -2.991311 ± 0.000056 [σ²=0.000000, R̂=1.4135]\n",
+      "300 -2.991311 ± 0.000056 [σ²=0.000000, R̂=1.4135]\n",
+      "350 -2.991311 ± 0.000056 [σ²=0.000000, R̂=1.4135]\n",
+      "400 -2.994530 ± 0.000011 [σ²=0.000000, R̂=1.4135]\n",
+      "450 -2.994530 ± 0.000011 [σ²=0.000000, R̂=1.4135]\n"
```

Comment:
why the outputs are so different with respect to the old tutorial?

Reply:
they aren't anymore.
You were using a complex network before and i had a real one.
+ the initialisation thing